testdata/go/v2/memcached-operator/PROJECT

```diff
@@ -8,4 +8,5 @@ resources:
   version: v1alpha1
 version: 3-alpha
 plugins:
-  go.sdk.operatorframework.io/v2-alpha: {}
```

Comment:
Here we are changing not adding. So, we might need to a specific entry for go/v2 ( change ) and for helm/ansible an additional.